components/ImageSlider.js

```diff
@@ -0,0 +1,104 @@
+import styled from "styled-components";
+
+export default function ImageSlider() {
+  return (
+    <>
+      <div>
```

Comment:
You don't need a <div> for this case. Your tags are already surrounded by a Fragment `<>`.

Reply:
thank you for the hint